x-pack/legacy/plugins/task_manager/migrations.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export default {
+  task: {
+    '7.4.0': (doc) => {
+      doc.updated_at = new Date().toISOString();
```

Comment:
Migrations shouldn't mutate their arguments. (This is generally a good idea outside of migrations, too.)

So, this should be:

```js
return {
  ...doc,
  updated_at: new Date().toISOString(),
};
```